adoc/retail_chap_introduction.adoc

```diff
@@ -4,29 +4,29 @@
 include::entities.adoc[]
 
 
-
-[abstract]
---
+[[retail.sect.intro.what]]
+== What is {smr}?
 
 {smr} {productnumber} is an open source infrastructure management solution, optimized and tailored specifically for the retail industry.
+It uses the same technology as {susemgr}, but is customized to address the needs of retail organizations.
 
-This document provides an overview of {smr}, and guides you through initial installation and setup.
-It should be read in conjunction with the {susemgr} documentation suite, available from https://www.suse.com/documentation/suse-manager-3/
+It is designed for use in retail situations where customers can use point-of-service terminals to purchase or exchange goods, take part in promotions, or collect loyalty points.  
```

Comment:
I think this sentence is not needed. One of the usages can very well be school computers, or self-service kiosks in banks, hospitals, etc. (as was in old SLEPOS). I don't really like limiting our use cases again and again just and only for retail, despite the name.

Reply:
Then add these other use cases.

And to avoid limiting, say "for example" or "among others".

I feel this sentence is very much needed.

Reply:
Agree, I think it's a good representation of use cases which is otherwise lacking in the docs (and always the number one thing readers ask for!). I've added a para to elaborate.